src/statscollector/DefaultStatsCollector.ts

```diff
@@ -353,7 +359,9 @@ export default class DefaultStatsCollector implements StatsCollector {
    */
 
   private isStreamRawMetricReport(type: string): boolean {
-    return type === 'ssrc' || type === 'inbound-rtp' || type === 'outbound-rtp';
+    return ['inbound-rtp', 'outbound-rtp', 'remote-inbound-rtp', 'remote-outbound-rtp'].includes(
+      type
```

Comment:
There are two kinds of `ssrc`:
* key of a property
* value of property `type`

For example:
```
{
  ssrc: 12345, // serve as an ID
  type: 'ssrc, // identify the category of the report
  ...
}
```

This function servers as a filter to select the stream related metrics categories (`type`) from the original report. In the report returned by legacy `getStats` API, `ssrc` both a `type` and `property` and it's stream related. However in the standardized `getStats` report, the `ssrc` is no longer one of the `type` of the report but the it is still a property in the stream related reports.

So the message you called out is talking about the property `ssrc` instead of `type` ssrc. So it does not apply to this case. For the property `ssrc` SDK is stilling save them.

Reply:
What are the stats we get from `remote-outbound-rtp` and `remote-inbound-rtp`? Previously we ignore them right?

Reply:
They provide metrics for jitter, packetsLost, packetsReceived and roundTripTime.

We did ignore them previously, and I assume at the time Kyu implemented the stats collector, the `remote-*boud-rtp` categories are not implemented yet. Cause they are proposed at Auguest 2019. 

Anyway we need them to calculate roundTripTime.